api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestCustomFilter.java

```diff
@@ -311,4 +311,26 @@ public TensorsData invoke(TensorsData in) {
             /* expected */
         }
     }
+
+    @Test
+    public void testRegisterPreservedName_n() {
+        try {
+            CustomFilter.registerCustomFilter("auto",
```

Comment:
Even if tensor-filter subplugin "auto" is prohibited, a custom plugin "auto" should be allowed. If it's customfilter "auto", its subplugin name will be "custom" or "custom-cpp" or "custom-easy" or so.

Reply:
custom-filter in java is not a .so file (run without model file). It runs without so, and the fw name would be auto in this case.

Reply:
So, it is a custom filter with "auto" as its custom filter name?

What if we have two java custom filters in a single pipeline, then?

Reply:
If developer needs two custom-filters, implement two interfaces with different names and register them.
```
CustomFilter.registerCustomFilter("custom1", new CustomFilter.CustomFilterCallback() { ..
CustomFilter.registerCustomFilter("custom2", new CustomFilter.CustomFilterCallback() { ..

pipeline = "... ! tensor_filter framework=custom1 ! tensor_filter framework=custom2 ! ..."
```